Editor/EditorCore/ProBuilderEditorShortcuts.cs

```diff
@@ -43,5 +52,19 @@ static void Toggle_SelectMode()
 			if(ProBuilderEditor.instance != null)
 				ProBuilderEditor.instance.ToggleSelectionMode();
 		}
+
+		static bool CheckAndEnterPBContextIfNeeded()
+		{
+			if (ToolManager.activeContextType == typeof(GameObjectToolContext))
```

Comment:
I wonder if that should be triggerable from any context as long as a PB GO is selected (to check with Gabe)?
Like : "I have GOs with Splines and PB components, I'm in the spline context and want to directly enter Vertex edit mode" ?
```suggestion
			if (ToolManager.activeContextType != typeof(PositionToolContext))
```

Reply:
Checking with @gabrielw-us

Reply:
@lopezt-unity Applied the suggestion after confirming with Gabe.